runtime/layers/fpm-dev/Dockerfile

```diff
@@ -29,3 +28,7 @@ ENV PHP_INI_SCAN_DIR="/opt/bref/etc/php/conf.d:/var/task/php/conf.d:/var/task/ph
 # Run PHP-FPM
 # opcache.validate_timestamps=1 : cancels the flag in the base configuration so that files are reloaded
 CMD /opt/bin/php-fpm --nodaemonize --fpm-config /opt/bref/etc/php-fpm.conf -d opcache.validate_timestamps=1 --force-stderr
+
+FROM without_extensions
```

Comment:
Why the new `FROM` here? We would lose the EXPOSE, ENV and everything defined above?

Reply:
I've checked, `ENV`, `EXPOSE`, `CMD` & `ENTRYPOINT` will be inherited from the above, the only difference is that we won't copy extensions (xdebug, blackfire).

Reply:
Ohh I think I see it now, thank you!